repro.in

```diff
@@ -12,8 +12,9 @@ BUILDDIRECTORY=/var/lib/repro
 
 KEYRINGCACHE="${BUILDDIRECTORY}/keyring"
 
-BOOTSTRAPMIRROR="https://europe.mirror.pkgbuild.com/iso/latest"
-readonly bootstrap_img=archlinux-bootstrap-"$(date -u +%Y.%m)".01-"$(uname -m)".tar.gz
+IMG_RELEASE_TIMESTAMP="$(date -d '24 hours ago' -u +%Y.%m)".01
```

Comment:
out of curiosity as its not mentioned anywhere: whats the rational of using a 24h old bootstrap image? (maybe worth adding as commit description)

Reply:
I've added this commit message:
```
    This works around a race condition when archlinux-repro is started at
    2022-04-01 00:01 and tries to download the new bootstrap image image before the
    image is generated and distributed to all mirrors.
```

The CI for #111 was failing because of this:
```
gpg: WARNING: unsafe permissions on homedir '/var/lib/repro/_gnupg'
gpg: keybox '/var/lib/repro/_gnupg/pubring.kbx' created
gpg: /var/lib/repro/_gnupg/trustdb.gpg: trustdb created
gpg: key 7F2D434B9741E8AC: public key "Pierre Schmitz <pierre@archlinux.de>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: no ultimately trusted keys found
pub   rsa2048 2011-04-10 [SC]
      4AA4767BBC9C4B1D18AE28B77F2D434B9741E8AC
uid           [ unknown] Pierre Schmitz <pierre@archlinux.de>
sub   rsa2048 2011-04-10 [E]

==> Downloading bootstrap image...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 
Error: Process completed with exit code 22.
```

See #archlinux-reproducible on irc :)

Reply:
Sounds good, I just think context about *why* something is changed is preferably also available in commit messages for future readers :cat: